internal/sourcemap/caching.go

```diff
@@ -61,13 +61,13 @@ func NewCachingFetcher(
 	}
 
 	go func() {
-		for i := range invalidationChan {
-			c.Remove(i)
+		for identifier := range invalidationChan {
+			lruCache.Remove(identifier)
```

Comment:
I believe the variable name changes were triggered by my comment about the `bbb` variable. My mistake, I didn't intend to make you change all variable names to something meaningful. It is perfectly fine to use something like `i` in a loop. It is also fine to use `c` for a `cache`, if the scope of the variable is a couple of lines within the same method. 

Using `b`, `d` and `bbb` for `body`, `decodedBody` and `uncompressedBody` was already a bit harder to follow. IMO there it makes sense to be a bit more descriptive.

Reply:
No worries, I changed `i` because it is usually used for int/index so it wasn't obvious to me it was referring to identifiers.
For cache, I was worried `c` might be confused with the `CachingFetcher` so I made it explicit.

I don't have a strong opinion about it, let me know if I should revert the change :bow:

Reply:
no need to revert anything, thanks for the changes.